app/cbt/main.py

```diff
@@ -32,7 +32,7 @@ async def on_ready():
 
 @client.event
 async def on_message(message: discord.Message):
-    logger.info(f"Message from {message.author}: {message.content}")
+    logger.debug(f"Message from {message.author}: {message.content}")
```

Comment:
This change does not appear to be in scope, given the title of the PR.

Reply:
cry